t/90h2olog.t

```diff
@@ -69,6 +69,34 @@ subtest "h2olog", sub {
   ok is_uuidv4($h3s_accept->{"conn-uuid"}), "h3s-accept has a UUIDv4 field `conn-uuid`"
 };
 
+subtest "h2olog -t", sub {
+  my $tracer = H2ologTracer->new({
+    pid => $server->{pid},
+    args => [
+      "-t", "h2o:send_response_header",
+      "-t", "h2o:receive_request_header",
+      "-t", "h2o:h3s_destroy",
+    ],
+  });
+
+  my ($headers, $body) = run_prog("$client_prog -3 https://127.0.0.1:$quic_port/");
+  like $headers, qr{^HTTP/3 200\n}, "req: HTTP/3";
+
+  my $trace;
+  until (($trace = $tracer->get_trace()) =~ m{"h3s-destroy"}) {}
+
+  if ($ENV{H2OLOG_DEBUG}) {
+    diag "h2olog output:\n", $trace;
+  }
+
+  my %group_by;
+  foreach my $event(map { decode_json($_) } split /\n/, $trace) {
```

Comment:
```suggestion
  for my $event (map { decode_json($_) } split /\n/, $trace) {
```
I'm asking this because it's you, but do we want to use foreach instead of for?

Reply:
I use `foreach` in the form of `foreach my (LIST)`, and `for` in C-style for statements.  However, as you may know, `foreach` is just an alias to `for` in Perl, and I see them in the same way as Perl parser does, so I don't really distinguish the two keywords.

Reply:
Thanks. Based on the understanding that they are interchangeable, I had recalled that people were in favor of using `for` throughout. But I could well be wrong, and we see both `for my ...` and `foreach my ...` already used in our code base. Therefore, I'm fine with going with `foreach`.

Maybe just add a whitespace after `$event` and call it a day?